ecs-init/docker/docker.go

```diff
@@ -276,6 +281,13 @@ func (c *Client) getHostConfig() *godocker.HostConfig {
 		config.CacheDirectory() + ":" + config.CacheDirectory(),
 		config.CgroupMountpoint() + ":" + DefaultCgroupMountpoint,
 	}
+
+	// for al, al2 add host ssl cert directory mounts
+	if config.HostCertsDirPath() != "" {
```

Comment:
nit: save the Stats(), `if certdir := config.HostCertsDirPath(); certdir != "" { }`